qoffer-domain/src/main/groovy/life/qbic/portal/portlet/offers/create/CreateOfferInput.groovy

```diff
@@ -9,4 +11,17 @@ package life.qbic.portal.portlet.offers.create
  */
 interface CreateOfferInput {
 
-}
\ No newline at end of file
+  /**
+   * Saves an offer in a (persistent) datasource.
+   *
+   * Developers shall call this method to pass offer content
+   * provided from the user in order to trigger the completion
+   * of the business use case `Create Offer`,
+   * which will apply business policies for offer creation and storage
+   * in a pre-configured, optimally persistent data-source.
+   *
+   * @param offer {@link life.qbic.datamodel.dtos.business.Offer}
+   */
+  void saveOffer(Offer offer)
```

Comment:
I am not sure if I like that we have 2x a saveOffer method. Why is the use case then called create offer?